tests/unit/oauthtest.py

```diff
@@ -0,0 +1,128 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
nit: rename as `test_oauth.py`?

Reply:
`conftest` is special actually. It comes from `pytest` framework. `oauth_test_utils` is better - thanks.